src/components/component-card/ComponentCard.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactNode } from 'react';
+import { IconTags } from '../../icons';
+import { ComponentImage } from '../component-image';
+import { SComponentCard } from './styles/SComponentCard';
+import { SComponentCardDescription } from './styles/SComponentCardDescription';
+import { SComponentCardTitle } from './styles/SComponentCardTitle';
+import { STagContainer } from './styles/STagContainer';
+import { STagText } from './styles/STagText';
+
+interface IComponentCard {
+  title?: string | undefined;
+  description?: string | undefined;
+  tags?: string | undefined;
+  image?: ReactNode;
+}
+
+export const ComponentCard: React.FC<IComponentCard> = ({ title, description, tags, image }) => {
+  return (
+    <SComponentCard>
```

Comment:
@steiningermona I just realized that the card is clickable. It should receive an event handler for the on click.